handlers/bin/command-not-found

```diff
@@ -68,9 +68,10 @@ def check_installed(term, pkg, path):
 def find_package_by_file(term):
     ts = rpm.TransactionSet()
     mi = ts.dbMatch('basenames', term)
-    for i in mi:
-        return i['name']
-    return None
+    try:
+        return next(mi)['name'].decode()
+    except StopIteration:
+        return None
```

Comment:
I understand the code, but I would go only adding the `.decode()` inside the `for`. Managing the iteration protocol is not very clear for me, but I agree that is explicit that there is only one result expected or zero

Reply:
No hard complain from my side, tho